### DISCUSSION TITLE
Podcast playlist RSS items in seemingly random order

### DISCUSSION THREAD

**Discussion by mainiomooses** [AzuraCast/AzuraCast#7636] [General]
Hi.

The title pretty much says it all, the RSS feed items seem to be randomly ordered. Of course there is a basis for their order being what it is, but I can't figure out what it is.

[A link to a playlist feed for example](https://radio.opendoors.fi/public/1/podcast/1efbd47b-5e1b-6648-a470-f30f4b8d2016/feed)

I've tried to search for discussion about this but found nothing. What is the order based on? Is there a "hidden" way to rearrange the feed by means of GET parameters etc?

Thanks.

**Comment by BusterNeece** [comment]
This should be fixed in the latest Rolling Release. It's sorted, as other API endpoints are, descending by `publish_at` date.
